examples/ui.apps/pom.xml

```diff
@@ -57,7 +57,18 @@
                 <artifactId>filevault-package-maven-plugin</artifactId>
                 <extensions>true</extensions>
                 <configuration>
-                    <packageType>application</packageType>
+                    <packageType>mixed</packageType>
```

Comment:
probably "container" is a better option than "mixed" which is discouraged?

Reply:
`container` does not allow content. We would need to create a `container` just for the bundles, which is not that helpful.